docs/serving/samples/hello-world/helloworld-scala/README.md

```diff
@@ -111,10 +111,10 @@ local Docker Repository.
 
 === "yaml"
 
-    Apply the [Service yaml definition](./helloworld-scala.yaml):
+    Apply the [Service yaml definition](./service.yaml):
 
     ```shell
-    kubectl apply --filename helloworld-scala.yaml
+    kubectl apply --filename service.yaml
```

Comment:
I'd change `--filesname` -> `-f` to make these consistent across the board, let's maybe add that to the style guide too?
We use `kubectl apply -f -` in the other bit re applying YAML files